docs/guides/migrations.md

```diff
@@ -7,6 +7,27 @@ kb_sync_latest_only
 
 # Migrations
 
+## From 6.0.0 to 7.0.0
+
+The Intershop PWA 7.0.0 release contains the **Sparque suggest** and **Sparque search** functionality to improve the searching of products in the PWA.
+To prevent compiling issues you have to consider the following changes:
+
+| Description              | API Changes                 | How to Migrate                                                                                                                                                                                                                    |
+| :----------------------- | :-------------------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| remove model             | _suggest-term.model.ts_     | Use the _keywordSuggestions_ parameter of the _suggestion.model.ts_ instead.                                                                                                                                                      |
+| remove service           | _suggest.service.ts_        | Method _search_ is renamed to _searchSuggestions_ and is now part of the _products.service.ts_. Method method <br> signature has changed to. The new return parameter is an Observable of <br> Suggestion(_suggestion.model.ts_). |
```

Comment:
What is meant by "Method method <br> signature has changed to."? Why is the line break needed?

Reply:
This part of the documentation will be rewritten and maybe be without a table. So we do not need to worry about the meaning right now. ;)